src/pkgcheck/checks/python.py

```diff
@@ -263,6 +279,14 @@ class PythonCheck(Check):
         "python-r1": "python_gen_any_dep",
     }
 
+    setuptool_scm = frozenset(
```

Comment:
```suggestion
    setuptools_scm = frozenset(
```

It looks weird to make it singular.